client/package.json

```diff
@@ -15,6 +15,7 @@
     "@fortawesome/fontawesome-free": "^5.15.4",
     "@material-ui/lab": "^4.0.0-alpha.60",
     "@mui/icons-material": "^5.4.1",
+    "@mui/lab": "^5.0.0-alpha.73",
```

Comment:
What is the difference between `@material-ui/lab` and `@mui/lab`? Should we be using one or the other rather than both, or are they truly separate things?

Reply:
Materia-ui lab is the depecrated version. We should be using the @mui/lab version.

Reply:
I'll make the change to move all references.

Reply:
Thanks! I think we're only using the old version in one place: for the tree-based model selection control in the "create object" component.

Reply:
Made change.